content/en/continuous_delivery/deployments/_index.md

```diff
@@ -24,14 +24,16 @@ CD Visibility is in private beta. To request access, complete the form.
 
 ## Overview
 
-[Deployment Visibility][1] provides a deployment-first view into your CD health by displaying important metrics and results from your deployments. 
+[Deployment Visibility][1] provides a deployment-first view into your CD health by displaying important metrics and results from your deployments.
 
 ## Setup
 
-{{< whatsnext desc="Select your CD provider to set up Deployment Visibility in Datadog:" >}}
-    {{< nextlink href="continuous_delivery/deployments/argocd" >}}ArgoCD{{< /nextlink >}}
+{{< whatsnext desc="Select your deployments provider to set up CD Visibility in Datadog:" >}}
+    {{< nextlink href="continuous_delivery/deployments/argocd" >}}Argo CD{{< /nextlink >}}
+    {{< nextlink href="continuous_delivery/deployments/ciproviders" >}}CI Providers (Gitlab, Buildkite, etc.){{< /nextlink >}}
```

Comment:
I have added another element in the list for CI providers deployments. Do we think this is an edge case and we should mention it separately instead (as it's not a CD provider but a list of CI Providers)?

Reply:
@andrea-mosk eventually we may want to call out each of our officially supported CI tools but for now I think a single link is fine.  Let's mention them in order of popularity like this:
> CI Providers ( Github actions, Gitlab, Jenkins, CircleCI, etc.)

Reply:
👍 Changed it